crates/router/src/connector/multisafepay/transformers.rs

```diff
@@ -237,8 +256,10 @@ impl TryFrom<&types::PaymentsAuthorizeRouterData> for MultisafepayPaymentsReques
         let description = item.get_description()?;
         let payment_options = PaymentOptions {
             notification_url: None,
-            redirect_url: item.request.router_return_url.clone(),
-            cancel_url: None,
+            redirect_url: Some(item.request.get_router_return_url()?)
+                .map(|return_url| format!("{return_url}?status=success")),
```

Comment:
Why we are hardcoding the status field in the URL? Are we taking any action based on this field?

Reply:
We hard coded the status field URL as multisafepay returns to the redirect_url only on a successful payment, for an unsuccessful payment it returns to cancel_url